src/webgpu/api/operation/resource_init/depth_stencil_attachment_clear.spec.ts

```diff
@@ -0,0 +1,242 @@
+export const description =
+  'Test uninitialized textures are initialized to zero when used as a depth/stencil attachment.';
+
+import * as C from '../../../../common/constants.js';
+import { TestGroup } from '../../../../common/framework/test_group.js';
+import { unreachable } from '../../../../common/framework/util/util.js';
+import {
+  fillTextureDataWithTexelValue,
+  getTextureCopyLayout,
+} from '../../../util/texture/layout.js';
+import { SubresourceRange } from '../../../util/texture/subresource.js';
+import { getTexelDataRepresentation } from '../../../util/texture/texelData.js';
+
+import {
+  InitializedState,
+  ReadMethod,
+  TextureZeroInitTest,
+  initializedStateAsDepth,
+  initializedStateAsStencil,
+} from './texture_zero_init_test.js';
+
+class DepthStencilAttachmentClearTest extends TextureZeroInitTest {
+  private depthTestPipelineCache: Map<string, GPURenderPipeline> = new Map();
```

Comment:
I didn't think about this before, but does this cache do anything? The whole fixture class gets constructed once for each test case, and anyway we can't rely on the device being the same between test cases.

We could cache the generated SPIR-V across cases. But since we're already not doing it, it doesn't seem extremely necessary.

Reply:
Ah welp. Removed the caching then. I'll remove it from the other test in another PR.